ADALiOS/ADALiOS/ADLogger.m

```diff
@@ -126,6 +126,8 @@ +(void) log: (ADAL_LOG_LEVEL)logLevel
                 sLogCallback(logLevel, [NSString stringWithFormat:@"ADALiOS [%@ - %@] %@", [dateFormatter stringFromDate:[NSDate date]], [[ADLogger getCorrelationId] UUIDString], message], additionalInformation, errorCode);
             }
         }
+        SAFE_ARC_RELEASE(dateFormatter);
```

Comment:
will it be okay if I put nil after the release here? I think it will require a major sweep across the code else we can get double nil assignment errors.